File: app/src/main/java/resource/com/br/santanderapp/model/investmentModel/Investment.java
package resource.com.br.santanderapp.model.investmentModel;

import com.google.gson.annotations.SerializedName;

public class Investment{

    @SerializedName("screen")
    private Screen screen;



    public Screen getScreen() {
        return screen;
    }

    public void setScreen(Screen screen) {
        this.screen = screen;
    }
}

File: app/src/main/java/com/fuinha11/test/testeandroid/data/model/Investment.kt
package com.fuinha11.test.testeandroid.data.model

import com.google.gson.annotations.SerializedName

data class Investment (
        val title: String,
        val fundName: String,
        val whatIs: String,
        val definition: String,
        val riskTitle: String,
        val risk: Int,
        val infoTitle: String,
        val info: List<Info>,
        val moreInfo: MoreInfo,
        val downInfo: List<Info>
)

data class Info (
        val name: String,
        val data: String
)

data class MoreInfo (
        val month: MoreInfoData,
        val year: MoreInfoData,
        @SerializedName("12months")
        val twelvemonths: MoreInfoData
)

data class MoreInfoData (
        val fund: Double,
        val CDI: Double
)